src/zenml/integrations/polars/__init__.py

```diff
@@ -0,0 +1,35 @@
+#  Copyright (c) ZenML GmbH 2023. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+"""Initialization of the Polars integration."""
+
+from zenml.integrations.constants import POLARS
+from zenml.integrations.integration import Integration
+
+
+class PolarsIntegration(Integration):
+    """Definition of Polars integration for ZenML."""
+
+    NAME = POLARS
+    REQUIREMENTS = [
+        "polars",
+        "pyarrow"
```

Comment:
this is probably too wide / open a specification for us :)

could probably be `^0.20.0` for polars and `^14.0.0` for pyarrow. WDYT? Or should we loosen the bottom edge a bit more and just specify < 0.21.0 and < 15.0 respectively?

Reply:
^12.0.0 wouldn't allow people to use >14.0 (I think we're currently on 14.2), and same for polars. But I suppose both are common / popular enough that maybe we should just specify a lower ceiling and set both to be `>=` to that ceiling? I think the versions you suggest sound fine to me for that threshold, though I say that without having a strong sense of whether this materializer actually is compatible etc.

Reply:
Ah, indeed, Friday afternoon mishap 😅
I just tested with those versions (`pip install polars==0.19.5 pyarrow==12.0.0` instead of the integration installs) and everything ran smoothly, so I figure we can safely add them.

Reply:
let's do it.

Reply:
Added in https://github.com/zenml-io/zenml/pull/2229/commits/536a5c52a8e459873379ff0ba8557d29a3ac3cca